packages/mdc-theme/_variables.scss

```diff
@@ -16,20 +16,28 @@
 
 @import "./functions";
 
-/*
-  Main theme colors.
-  If you're a user customizing your color scheme in SASS, these are probably the only variables you need to change.
-*/
-$mdc-theme-primary: #3f51b5 !default; /* Indigo 500 */
-$mdc-theme-accent: #ff4081 !default; /* Pink A200 */
-$mdc-theme-background: #fff !default; /* White */
+//
```

Comment:
Super nit: if we introduce no empty lines in other file, could we do consistently in this file?